From c2c371aa64609410342013d3f61b5b4a956cc4c4 Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Wed, 22 Jul 2015 22:57:05 +0800
Subject: [PATCH] MLK-11276-2 ARM: dts: correct code indent for gpmi nand node

Correct the code indent to make code more beautiful and
follow the coding style.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 arch/arm/boot/dts/imx7d.dtsi | 28 ++++++++++++++--------------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index 61df71a..7e2658f 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -376,21 +376,21 @@
 			clock-names = "dma_apbh_bch", "dma_apbh_io";
 		};
 
-        gpmi: gpmi-nand@33002000{
+		gpmi: gpmi-nand@33002000{
 			compatible = "fsl,imx7d-gpmi-nand";
-            #address-cells = <1>;
-            #size-cells = <1>;
-            reg = <0x33002000 0x2000>, <0x33004000 0x4000>;
-            reg-names = "gpmi-nand", "bch";
-            interrupts = <GIC_SPI 14 IRQ_TYPE_LEVEL_HIGH>;
-            interrupt-names = "bch";
-            clocks = <&clks IMX7D_NAND_ROOT_CLK>,
-                 <&clks IMX7D_NAND_USDHC_BUS_ROOT_CLK>;
-            clock-names = "gpmi_io", "gpmi_bch_apb";
-            dmas = <&dma_apbh 0>;
-            dma-names = "rx-tx";
-            status = "disabled";
-        };
+			#address-cells = <1>;
+			#size-cells = <1>;
+			reg = <0x33002000 0x2000>, <0x33004000 0x4000>;
+			reg-names = "gpmi-nand", "bch";
+			interrupts = <GIC_SPI 14 IRQ_TYPE_LEVEL_HIGH>;
+			interrupt-names = "bch";
+			clocks = <&clks IMX7D_NAND_ROOT_CLK>,
+				<&clks IMX7D_NAND_USDHC_BUS_ROOT_CLK>;
+			clock-names = "gpmi_io", "gpmi_bch_apb";
+			dmas = <&dma_apbh 0>;
+			dma-names = "rx-tx";
+			status = "disabled";
+		};
 
 		aips1: aips-bus@30000000 {
 			compatible = "fsl,aips-bus", "simple-bus";
-- 
1.8.0

